src/objects/Player.java

```diff
@@ -0,0 +1,68 @@
+package objects;
+
+import java.util.Arrays;
+
+public class Player {
+	private Card[] hand;
+	private int playerID;
```

Comment:
Since we have a player array (in the GUI), having a playerID field seems redundant

Reply:
done.